k8s/vpa.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: bankapp-vpa
+  namespace: bankapp-namespace
+spec:
+  targetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: bankapp-deployment
+  updatePolicy:
+    updateMode: "Auto" # Options: "Off", "Initial", "Auto"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)